orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/MonitorWebhookTask.groovy

```diff
@@ -38,6 +38,7 @@ class MonitorWebhookTask implements OverridableTimeoutRetryableTask {
 
   long backoffPeriod = TimeUnit.SECONDS.toMillis(1)
   long timeout = TimeUnit.HOURS.toMillis(1)
+  private static final String JSON_PATH_NOT_FOUND_ERR_FMT = "Unable to parse status: JSON property '%s' not found in response body"
```

Comment:
Not expert expect but looks like this error message is used for status and progress path failures ... perhaps just `JSON property '%s' not found in response body` would be sufficient?

Either way, much better than the exception message I reckon.

Reply:
Ah, you're right, thank you!

Reply:
Made it parametrized. I think the `status` vs. `progress` is useful distinction